WordPress/src/main/java/org/wordpress/android/ui/reader/views/compose/filter/ReaderFilterChipGroup.kt

```diff
@@ -97,17 +99,19 @@ fun ReaderFilterChipGroup(
         }
 
         // tags filter chip
-        AnimatedVisibility(
-            modifier = Modifier.clip(roundedShape),
-            visible = isTagChipVisible,
-        ) {
-            ReaderFilterChip(
-                text = tagChipText,
-                onClick = if (isTagSelected) onSelectedItemClick else ({ onFilterClick(ReaderFilterType.TAG) }),
-                onDismissClick = if (isTagSelected) onSelectedItemDismissClick else null,
-                isSelectedItem = isTagSelected,
-                height = chipHeight,
-            )
+        if (showTagsChip) {
```

Comment:
Approving this PR as it works as expecred. 
I have a question regarding the use of a feature flag for this feature. Is it really necessary?  Given that we're developing the feature in a feature branch, and will likely be enabled for all users soon after approval. If a feature flag is necessary, we should remember to clean up the code immediately after merging into the trunk to avoid adding conditional statements to the code.

Reply:
Good question. I was discussing this with @wargcm yesterday, and I implemented it on Android since we also have it on iOS.

> we should remember to clean up the code immediately after merging into the trunk to avoid adding conditional statements to the code.

I agree.